test_vm/scripts/postgresql/sql/create-db-schema-postgresql.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS changelog (
```

Comment:
Perhaps that script can be reused from the postgres Chef recipe to avoid duplication.